backend/.env.docker

```
@@ -0,0 +1,9 @@
+MONGODB_URI="mongodb://mongo-service/wanderlust"
+REDIS_URL="redis://redis-service:6379"
+PORT=5000
+ACCESS_COOKIE_MAXAGE=120000
+ACCESS_TOKEN_EXPIRES_IN='120s'
+REFRESH_COOKIE_MAXAGE=120000
+REFRESH_TOKEN_EXPIRES_IN='120s'
+JWT_SECRET=70dd8b38486eee723ce2505f6db06f1ee503fde5eb06fc04687191a0ed665f3f98776902d2c89f6b993b1c579a87fedaf584c693a106f7cbf16e8b4e67e9d6df
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the MONGODB_URI key

(UnorderedKey)